### ISSUE TITLE
Internal server error for Pipelines / Tasks / ClusterTasks page using All Namespaces

### ISSUE THREAD

**Issue by CarolynMabbott** [tektoncd/dashboard#1472]
# Expected behavior
It doesnt error and shows me info

# Actual behavior
It errors and i see no info on the pages 

# Steps to reproduce the problem

1. Go port pipelines / tasks / clustertasks page when All Namespaces on the left
2.
3.

# Environment
- Kubernetes Platform:

Openshift 

- Tekton Pipelines version:

0.11.3

- Tekton Triggers version:

0.4.0

- Tekton Dashboard version:

0.7.0
# Additional Info
Installed pipelines / triggers through operator 

![Screenshot 2020-06-05 at 13 40 48](https://user-images.githubusercontent.com/32342288/83877411-69981780-a732-11ea-808c-19735ab55382.png)

In the logs
```
{"level":"error","msg":"Error: {kind:Status,apiVersion:v1,metadata:{},status:Failure,message:conversion webhook for tekton.devv1beta1, Kind=PipelineList failed: Post https:tekton-pipelines-webhook.tekton-pipelines.svc:443resource-conversion?timeout=30s: service tekton-pipelines-webhook not found,code:500}"}
{"level":"error","msg":"Error: {kind:Status,apiVersion:v1,metadata:{},status:Failure,message:conversion webhook for tekton.devv1beta1, Kind=PipelineList failed: Post https:tekton-pipelines-webhook.tekton-pipelines.svc:443resource-conversion?timeout=30s: service tekton-pipelines-webhook not found,code:500}"}
{"level":"error","msg":"Error: {kind:Status,apiVersion:v1,metadata:{},status:Failure,message:conversion webhook for tekton.devv1beta1, Kind=TaskList failed: Post https:tekton-pipelines-webhook.tekton-pipelines.svc:443resource-conversion?timeout=30s: service tekton-pipelines-webhook not found,code:500}"}
{"level":"error","msg":"Error: {kind:Status,apiVersion:v1,metadata:{},status:Failure,message:conversion webhook for tekton.devv1beta1, Kind=ClusterTaskList failed: Post https:tekton-pipelines-webhook.tekton-pipelines.svc:443resource-conversion?timeout=30s: service tekton-pipelines-webhook not found,code:500}"}
{"level":"error","msg":"Error: {kind:Status,apiVersion:v1,metadata:{},status:Failure,message:conversion webhook for tekton.devv1beta1, Kind=TaskList failed: Post https:tekton-pipelines-webhook.tekton-pipelines.svc:443resource-conversion?timeout=30s: service tekton-pipelines-webhook not found,code:500}"}
```
/cc @AlanGreene

**Comment by AlanGreene** [comment]
`message:conversion webhook for tekton.devv1beta1`

Shouldn't that be `tekton.dev/v1beta1` (notice the missing '/')

Not sure if that's just a display issue in the logs you posted or pointing at a possible cause?

**Comment by CarolynMabbott** [comment]
I have everything installed in the same namespace
installed pipelines + triggers through operator 
Installed latest dashboard release through oc apply 

That missing slash could be the problem, it's exactly the same as in the logs

**Comment by CarolynMabbott** [comment]
So an update, it works fine when I access pipelines and tasks when any namespace is selected on the left but when its All Namespaces it errors
Clustertasks is still broken

**Comment by a-roberts** [comment]
Carolyn's seeing this with 0.7.0 which is cut from master a few days ago, and master (which I staged a release candidate for) - holding up the release for that until this is resolved

**Comment by a-roberts** [comment]
@CarolynMabbott I don't see this with the latest release candidate or the staged one on OpenShift - do you? Can you check everything looks ok in your cluster?

**Comment by AlanGreene** [comment]
Hasn't been reported by anyone else and we've since released 0.7.0. Feel free to reopen if you do encounter it again.
